docs/commands/preview.md

```diff
@@ -14,19 +14,19 @@ This command is for our pre-release products, currently open for early access to
+| --product         | string  | Name of a product to preview the project with. <br/> **Possible values:** `redoc`, `revel`, `reef`, `realm`, `redoc-revel`, `redoc-reef`, `revel-reef`. <br/> `redoc` is the flagship product for generating API documentation from OpenAPI specifications. <br/> `revel` is a specialized product designed for external API applications. <br/> `reef` is a specialized product designed for internal API needs. <br/> `realm` is a balanced product combining `redoc`, `revel`, and `reef`. <br/> `redoc-revel` is a blended product combining `redoc` and `revel`. <br/> `redoc-reef` is a blended product combining `redoc` and `reef`. <br/> `revel-reef` is a blended product combining `revel` and `reef`. <br/> The default value is autodetected from the project's `package.json` or `realm` is used. |
+| --project-dir, -d | string  | Path to the project directory. The default value is `.` (current directory).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| --port, -p        | number  | The port to run the preview server on. The default value is `4000`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
This may not be a helpful change for a command that has port, plan and product parameters. Not a blocking request for change, but a general note that we did discuss this at the time that the command was implemented.

Reply:
I don't see the problem with that at all.

Here is an example of a `psql` (Postgress client) CLI docs: 
![image](https://github.com/user-attachments/assets/3e4503e6-3ebc-41c2-b3ba-03950eda5136)

They have multiple args that start with letter `p` but still use `-p` as `--port`.

I also see this very common to in linux CLI tools, e.g. `curl -i` where -i is an alias for `--show-headers` while curl has many other arguments starting with letter `i`: `--include`, `--insecure`, `--interface`.